lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -85,33 +85,87 @@ void notifyWithoutExceptions(const NOTIFICATION& notif)
 // use a counter nonetheless to provide consistent behavior in such cases.
 std::atomic_int attributeChangedNotificationGuardCount { 0 };
 
-// Keep an array of the pending attribute change notifications. Using a vector
+enum class AttributeChangeType
+{
+    added,
+    valueChanged,
+    connectionChanged,
+    removed
```

Comment:
According to our coding standard these should be named like constants with "kXxxx"